components/form/__tests__/__snapshots__/demo.test.js.snap

```diff
@@ -1337,6 +1337,7 @@ exports[`renders ./components/form/demo/normal-login.md correctly 1`] = `
                 class="ant-checkbox-input"
                 data-__field="[object Object]"
                 data-__meta="[object Object]"
+                id="remember"
```

Comment:
How does this come from?

Reply:
I find the reason https://github.com/react-component/checkbox/commit/579d2cc20c7d703affddebd40c0e2839e10da855, this is expected.